app/components/wheel-animations.hbs

```diff
@@ -0,0 +1,25 @@
+<section class='animation__container' data-test-wheel-animation-section>
+  {{#if @users}}
+    <div>
+      {{#each @users as |m index|}}
+        <a
+          href='https://members.realdevsquad.com/{{m.username}}'
+          target='_blank'
+          class='member__link'
+          rel='noopener noreferrer'
+          data-test-wheel-animation-member-link
+        >
+          <img
+            style={{(this.memberStyle index)}}
+            class='circle member__img'
+            src='{{(this.resizeImageUrl m.picture.publicId 150 150)}}'
+            alt='{{m.first_name}} {{m.last_name}}'
+            data-test-wheel-animation-member-image
```

Comment:
`data-test-wheel-animation-member-image={{somedynamicvaluefrom<mobject>>}}`

Reply:
I don't think there is any need for this, as when testing the content length and single elements links are checked to be enough, so there is no need to add a number, as there is no need to iterate over the same test

Reply:
okay